protocols/identify/src/handler.rs

```diff
@@ -82,6 +114,12 @@ pub struct Handler {
         >; 4],
     >,
 
+    /// Streams awaiting `BehaviourInfo` to then send identify requests.
+    reply_streams: VecDeque<ReplySubstream<NegotiatedSubstream>>,
```

Comment:
Is the abstraction of a `ReplySubstream` useful here? Would it be easier to just buffer the streams themselves?

Reply:
Sorry Thomas, not following. What is the alternative? But the `send` method is implemented for `ReplySubstream`.

Reply:
The alternative is inlining `send` to wherever it is called which I'd guess is only one place?

Reply:
We don't really do this wrapping of a substream anywhere else in the codebase so it would be nice to align it here.

Wrapping the stream in a `Framed` is what we typically do for other protocols!

Reply:
ah, right! Thanks! I was able to remove `ReplySubstream` but `send` is still being used in more places. 
We can go further and deprecate `Info` and just use the protobuf `Identify`, but probably do that on a subsequent PR?

Reply:
> ah, right! Thanks! I was able to remove `ReplySubstream` but `send` is still being used in more places. 
> We can go further and deprecate `Info` and just use the protobuf `Identify`, but probably do that on a subsequent PR?

Nah, I think that is fine. We usually don't use the protobuf struct directly which is good.